react-components/src/desktop/view/Notifications.js

```diff
@@ -62,7 +64,47 @@ function NotificationFooter(props) {
                 </span>
             }
         </MenuItem>
-    )
+    );
+}
+
+function Notification(props) {
+    const {notification, onClick} = props;
+    let notificationStyle = {
+        fontSize: 10,
+    };
+    if (!notification.seen) {
+        notificationStyle.backgroundColor = '#99d9ea';
+        notificationStyle.borderBottom = 1;
+    }
+
+    return (
+        <span style={{outline: 'none'}}>
+                    <MenuItem
+                        id={notification.message.id}
+                        onClick={onClick}
+                        style={notificationStyle}>
```

Comment:
Instead of defining the styles here, could we move these to the styles file?  Maybe we could have something like

```
seenNotification: {
    fontSize: 10
},
unseenNotification: {
   fontSize: 10,
   backgroundColor: '#99d9ea',
   borderBottom: 1
}
```

And then on this line, we could have:
```
<MenuItem
   id={notification.message.id}
   onClick={onClick}
   className={notification.seen ? classes.seenNotification : classes.unseenNotification}>
```